src/form-elements/FormElementText.tsx

```diff
@@ -86,6 +87,7 @@ function FormElementText({
             <div className="is-flex is-justify-content-space-between">
               {isDisplayingValidationMessage ? (
                 <div className="has-text-danger ob-error__text cypress-validation-message">
+                  <ValidationIcon />
                   {validationMessage}
                 </div>
```

Comment:
Is it worth making a component for just this?

Reply:
It's only used in two places and unlikely to ever be a third.  I'm ok not to.